docs/ref/forms/models.txt

```diff
@@ -70,12 +70,12 @@ Model Form API reference. For introductory material about model forms, see the
 
     .. versionchanged:: 3.2
 
-        The ``absolute_max`` argument was added.
+        The ``absolute_max`` and ``can_delete_extra`` arguments were added.
```

Comment:
We should add `can_delete_extra` to [the above list of arguments](https://github.com/django/django/pull/13097/files#diff-ad82f031e7988d81829bb4cf25a58dbbR64-R67), e.g.
```diff
diff --git a/docs/ref/forms/models.txt b/docs/ref/forms/models.txt
index e18660b3c2..aec3d977a3 100644
--- a/docs/ref/forms/models.txt
+++ b/docs/ref/forms/models.txt
@@ -62,8 +62,9 @@ Model Form API reference. For introductory material about model forms, see the
     through to :func:`~django.forms.models.modelform_factory`.
 
     Arguments ``formset``, ``extra``, ``max_num``, ``can_order``,
-    ``can_delete``, ``validate_max``, and ``absolute_max`` are passed through
-    to :func:`~django.forms.formsets.formset_factory`. See :doc:`formsets
+    ``can_delete``, ``can_delete_extra``, ``validate_max``, and
+    ``absolute_max`` are passed through to
+    :func:`~django.forms.formsets.formset_factory`. See :doc:`formsets
     </topics/forms/formsets>` for details.
 
     See :ref:`model-formsets` for example usage.
```